solution/ui/regulations/utilities/utils.js

```diff
@@ -738,6 +736,40 @@ function trapFocus(element) {
     };
 }
 
+/**
+ * Convert digit to roman numeral.
+ * Adapted from: https://blog.stevenlevithan.com/archives/javascript-roman-numeral-converter#comment-823972
+ *
+ * @param {number} num - number to convert
+ * @returns {string} - roman numeral
+ */
+const romanize = (num) => {
```

Comment:
this is not currently being used due to edits made to the caption, but seems useful enough to keep around for now